packages/grafana-ui/src/components/BigValue/BigValueLayout.tsx

```diff
@@ -130,17 +130,24 @@ export abstract class BigValueLayout {
 
     const themeFactor = theme.isDark ? 1 : -0.7;
 
+    function buildGradientBackground(valueColor: BigValueLayout['valueColor']) {
+      const bgColor2 = tinycolor(valueColor)
+        .darken(15 * themeFactor)
+        .spin(8)
+        .toRgbString();
+      const bgColor3 = tinycolor(valueColor)
+        .darken(5 * themeFactor)
+        .spin(-8)
+        .toRgbString();
+
+      return `linear-gradient(120deg, ${bgColor2}, ${bgColor3})`;
+    }
+
     switch (colorMode) {
       case BigValueColorMode.Background:
-        const bgColor2 = tinycolor(this.valueColor)
-          .darken(15 * themeFactor)
-          .spin(8)
-          .toRgbString();
-        const bgColor3 = tinycolor(this.valueColor)
-          .darken(5 * themeFactor)
-          .spin(-8)
-          .toRgbString();
-        panelStyles.background = `linear-gradient(120deg, ${bgColor2}, ${bgColor3})`;
+        hasGradient
```

Comment:
using it this way change the behavior of existing panels (that do not have the value set and will this will evaluate to false!)

Perhaps use something like
```suggestion
        (hasGradient == false) ? ... : ...
```
so that it has to be explicitly set and saved as false